helm/azure-operator/templates/configmap.yaml

```diff
@@ -15,69 +15,70 @@ data:
         address: 'http://0.0.0.0:8000'
     service:
       azure:
-        environmentName: '{{ .Values.Installation.V1.Provider.Azure.Cloud }}'
+        environmentName: '{{ .Values.azure.environmentName }}'
         hostCluster:
-          cidr: '{{ .Values.Installation.V1.Provider.Azure.HostCluster.CIDR }}'
-          resourceGroup: '{{ .Values.Installation.V1.Provider.Azure.HostCluster.ResourceGroup }}'
-          virtualNetwork: '{{ .Values.Installation.V1.Provider.Azure.HostCluster.VirtualNetwork }}'
-          virtualNetworkGateway: '{{ .Values.Installation.V1.Provider.Azure.HostCluster.VirtualNetworkGateway }}'
-        location: '{{ .Values.Installation.V1.Provider.Azure.Location }}'
+          cidr: '{{ .Values.azure.hostCluster.cidr }}'
+          resourceGroup: '{{ .Values.azure.hostCluster.resourceGroup }}'
+          virtualNetwork: '{{ .Values.azure.hostCluster.virtualNetwork }}'
+          virtualNetworkGateway: '{{ .Values.azure.hostCluster.virtualNetworkGateway }}'
+        location: '{{ .Values.azure.location }}'
         msi:
-          enabled: '{{ .Values.Installation.V1.Provider.Azure.MSI.Enabled }}'
+          enabled: {{ .Values.azure.msi.enabled }}
       cluster:
-        baseDomain: '{{ .Values.Installation.V1.BaseDomain }}'
+        baseDomain: '{{ .Values.cluster.baseDomain }}'
         calico:
-          cidr: '{{ .Values.Installation.V1.Guest.Calico.CIDR }}'
+          cidr: '{{ .Values.cluster.calico.cidr }}'
           mtu: 1430
-          subnet: '{{ .Values.Installation.V1.Guest.Calico.Subnet }}'
+          subnet: '{{ .Values.cluster.calico.subnet }}'
         docker:
           daemon:
-            cidr: '{{ .Values.Installation.V1.Guest.Docker.CIDR }}'
+            cidr: '{{ .Values.cluster.docker.daemon.cidr }}'
             extraArgs: '--log-opt max-size=25m --log-opt max-file=2 --log-opt labels=io.kubernetes.container.hash,io.kubernetes.container.name,io.kubernetes.pod.name,io.kubernetes.pod.namespace,io.kubernetes.pod.uid'
         etcd:
           altNames: ''
           port: 2379
           prefix: 'giantswarm.io'
         kubernetes:
           api:
-            altNames: 'kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.{{ .Values.Installation.V1.Guest.Kubernetes.API.Domain }}'
-            clusterIPRange: '{{ .Values.Installation.V1.Guest.Kubernetes.API.ClusterIPRange }}'
+            altNames: 'kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.cluster.local'
```

Comment:
Hardcoded??

Reply:
It was the same for each installation.

Reply:
what about the ip CIDR?

Reply:
`clusterIPRange` line 44

Reply:
This is because `.Values.Installation.V1.Guest.Kubernetes.API.Domain` is never re-configured from default.

Reply:
I can imagine this striking back evntually, but ok